LayoutTests/fast/css3-text/font-synthesis-bold-small-font-size-expected.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<style>
+@font-face {
+  font-family: Ahem;
+  src: url('/LayoutTests/resources/Ahem.ttf');
+  font-weight: 400;
+}
+div {
+  font-family: Ahem;
+  width: 80px;
+  font-size: 11px;
+  border: 1px solid violet;
+}
+.test {
+  font-weight: 600;
+  color: white;
+}
+</style>
+Test passes if block occupies only 1 line:
+<div class="test">a</div>
+</html>
```

Comment:
Could this be a WPT instead? (@nt1m, what do you think?)

Reply:
The exact amount of boldness for synthesis isn't specified, so it would be better to not make a WPT.